shark/shark_downloader.py

```diff
@@ -153,7 +153,11 @@ def get_git_revision_short_hash() -> str:
         import json
 
         dir_path = os.path.dirname(os.path.realpath(__file__))
-        src = os.path.join(dir_path, "..", "tank_version.json")
+        res_path = os.path.join(dir_path, "resources", "shark_version.json")
+        if os.path.exists(res_path):
+            src = res_path
+        else:
+            src = os.path.join(dir_path, "shark_version.json")
```

Comment:
you will need to use https://github.com/nod-ai/SHARK/blob/main/apps/stable_diffusion/src/utils/resources.py